### ISSUE TITLE
cannot encode urls in form list

### ISSUE THREAD

**Issue by mitchellsundt** [opendatakit/opendatakit#333]
Originally reported on Google Code with ID 332

```
What steps will reproduce the problem?
1. put the following form list on the server:

<forms>
  <form url="http%3A%2F%2Fwww.example.com%2Fgetform%3Fsomeparam%3D123%26otherparam%3D456">my
form</form>
</forms>

Note that the above url is just http://www.example.com/getform?someparam=123&otherparam=456
url-encoded

2. have odk collect get new forms. The form will appear in the list
3. try to get the form.

What is the expected output? What do you see instead?

Expected is that ODK Collect should decode the url and get the form from http://www.example.com/getform?someparam=123&otherparam=456.

Instead, it gives the following error:

Form ... failed. Protocol not found: http%3A%2F%2Fwww.example.com%2Fgetform%3Fsomeparam%3D123%26otherparam%3D456



If you don't url-encode the url, you'll get an error when retrieving the formList,
because the "&" inside the url attribute triggers a xml parsing error (which is expected).

So basically you cannot retrieve forms that have a "&" in the url.
```

Reported by `matteosistisette` on 2011-09-23 15:49:40
